ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -174,6 +175,25 @@ def negative_binomial(n, p, size=None):
     return ivy.poisson(lam=lambda_, shape=size)
 
 
+@with_supported_dtypes(
```

Comment:
I don't think you need `with_supported_dtypes` decorator for float64 dtype. You should remove this decorator 😄